kitsune/wiki/templates/wiki/document.html

```diff
@@ -83,6 +83,11 @@ <h1 class="product-title cf">
     {% set parent = doc.parent %}
   {% endif %}
   {{ document_tools(doc, parent, user, 'article', settings, include_showfor=True) }}
+  <h4>{{ _('Explore more topics') }}</h4>
   {{ topic_sidebar(product_topics[:10], [], product) }}
+  {% if related_products %}
+    <h4>{{ _('Related products') }}</h4>
```

Comment:
Can this header be a part of the macro, so we only have to include the if statement once? I don't care if the if statement goes inside or outside the macro (though I'd lean toward inside).